roles/contrail-ansible-deployer/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+docker_provision_dir: '{{ ansible_env.HOME }}/src/review.opencontrail.org/Juniper/contrail-ansible-deployer'
```

Comment:
review2, or we can change it to use zuul variables https://docs.openstack.org/infra/zuul/user/jobs.html#var-zuul.project.src_dir

Reply:
I'll change it to review2 now, switch to zuul later.